### PR TITLE
Add stow package

### PR DESCRIPTION
**Pull request by wwwiiilll**
Manage installation of multiple softwares in the same directory tree

### CODE REVIEW

**Comment by uberhacker** [comment]
@wwwiiilll: Thank you for the new package!  Can you please abstract the prefix with `system "./configure", "--prefix=#{CREW_PREFIX}"`?  Also, if there are any libraries installed those will need to be abstracted as well.  Something like `system "./configure", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}"`.
Thanks

File: packages/stow.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require 'package'
+
+class Stow < Package
+  description 'Manage installation of multiple softwares in the same directory tree'
+  homepage 'https://www.gnu.org/software/stow/'
+  version '2.2.2'
+  source_url 'http://ftp.gnu.org/gnu/stow/stow-2.2.2.tar.gz'
```
Comment (uberhacker) [inline packages/stow.rb]: Might be a good idea to use https instead of http.

**Comment by wwwiiilll** [comment]
Changed some stuff, I'll test it again later today, especially for the Perl modules stuff

**Comment by wwwiiilll** [comment]
Looks good now

**Pull request merged**